### ISSUE TITLE
E2E test test_pass_config failed

### ISSUE THREAD

**Issue by serhiy-storchaka** [neuromation/platform-client-python#1340]
https://dev.azure.com/neuromation/platform-client-python/_build/results?buildId=2350&view=logs&j=e7b3dc7a-ee4c-5376-13ef-6197b8c25284&t=0fb12b3b-6b58-50b5-0e6d-6c671e87df3c&l=414
```
_______________________________ test_pass_config _______________________________
[gw6] linux -- Python 3.6.10 /opt/hostedtoolcache/Python/3.6.10/x64/bin/python

image = 'neuro-cli-test:27f0cb1bb8122ec4ba1061ba690e88480329d9bcb801b005cd10851f3697556b'
helper = <tests.e2e.conftest.Helper object at 0x7fecf5d083c8>

    @pytest.mark.e2e
    def test_pass_config(image: str, helper: Helper) -> None:
        # Let`s push image
        captured = helper.run_cli(["image", "push", image])
    
        image_full_str = f"image://{helper.username}/{image}"
        assert captured.out.endswith(image_full_str)
    
        command = 'bash -c "neuro config show"'
        # Run a new job
        captured = helper.run_cli(
            [
                "job",
                "run",
                "-q",
                "-s",
                JOB_TINY_CONTAINER_PRESET,
                "--no-wait-start",
                "--pass-config",
                image_full_str,
                command,
            ]
        )
        job_id = captured.out
    
        # sleep(1)
    
        # Wait until the job is running
>       helper.wait_job_change_state_to(job_id, JobStatus.SUCCEEDED)

tests/e2e/test_e2e_jobs.py:703: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/e2e/conftest.py:94: in wrapper
    return run(_run_async(coro, *args, **kwargs))
neuromation/cli/asyncio_utils.py:116: in run
    return runner.run(main)
neuromation/cli/asyncio_utils.py:48: in run
    return self._loop.run_until_complete(main_task)
/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/asyncio/base_events.py:488: in run_until_complete
    return future.result()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

coro = <function Helper.wait_job_change_state_to at 0x7fecf6417b70>
args = (<tests.e2e.conftest.Helper object at 0x7fecf5d083c8>, 'job-c345b1ee-bb9e-4c5c-b65a-da2e6385a72c', <JobStatus.SUCCEEDED: 'succeeded'>)
kwargs = {}

    async def _run_async(
        coro: Callable[..., Awaitable[Any]], *args: Any, **kwargs: Any
    ) -> Any:
        try:
>           return await coro(*args, **kwargs)
E           AssertionError: timeout exceeded, last output: 'failed'

tests/e2e/conftest.py:84: AssertionError
```

**Comment by serhiy-storchaka** [comment]
Tests on master before 17.02.2020, 09:32:34 was passed.
Tests on master after 18.02.2020, 08:48:18 was failed.
There was no changes in client in this range.

**Comment by asvetlov** [comment]
Fixed by #1343
